optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/tenant/TenantRestServiceImpl.java

```diff
@@ -55,9 +55,12 @@ public Tenant addTenant(Tenant tenant) {
 
     @Override
     @Transactional
-    public Tenant updateTenantConfiguration(TenantConfigurationView tenantConfigurationView) {
-        Tenant tenant = entityManager.find(Tenant.class, tenantConfigurationView.getTenantId());
-        tenant.getConfiguration().setView(tenantConfigurationView);
+    public Tenant updateTenantConfiguration(TenantConfiguration tenantConfiguration) {
+        Tenant tenant = entityManager.find(Tenant.class, tenantConfiguration.getTenantId());
+        if (null == tenant) {
+            throw new IllegalStateException("There is no tenant with id " + tenantConfiguration.getTenantId());
```

Comment:
see my previous comment for this line that got removed?

Reply:
Odd; when I viewed it initially , it only showed "identation"; I see the second part now; will fix!